cvat-core/src/server-proxy.ts

```diff
@@ -2053,12 +2058,28 @@ async function deleteOrganizationMembership(membershipId: number): Promise<void>
     }
 }
 
-async function getMembershipInvitation(id) {
+async function getMembershipInvitations(
+    filter: InvitationsFilter = {},
+): Promise<{ results: SerializedInvitationData[], count: number }> {
     const { backendAPI } = config;
 
     let response = null;
     try {
-        response = await Axios.get(`${backendAPI}/invitations/${id}`);
+        const key = filter.key || null;
+
+        if (key) {
+            response = await Axios.get(`${backendAPI}/invitations/${key}`);
+            return ({
+                results: [response.data],
+                count: 1,
+            });
+        }
+
+        response = await Axios.get(`${backendAPI}/invitations`, {
+            params: {
+                ...filter,
```

Comment:
Do we need InvitationsFilter type? It has only one field ``key`` that is used above. Here is always empty object.

Reply:
Removed that.